scripts/upgradeNocodbSdk.js

```diff
@@ -5,6 +5,11 @@ const execSync = require('child_process').execSync;
 // extract latest version from package.json
 const nocodbSdkPackage = JSON.parse(fs.readFileSync(path.join(__dirname, '..', 'packages', 'nocodb-sdk', 'package.json'), 'utf8'))
 
+
+if (process.env.revertSDK === 'true') {
+  nocodbSdkPackage.version = 'workspace:^';
+}
+
```

Comment:
we have to update package version in `nocodb` and `nc-gui` dependencies

Reply:
version property in `nocodb-sdk/package.json` should remain same

Reply:
this script only updates nc-gui and nocodb package.json, nocodb-sdk version handled in bumpNocodbSdkVersion.js

Reply:
Oh sorry, here it's just reading and updating value in object.